common/output-model.cpp

```diff
@@ -981,13 +1040,14 @@ void stream_dashboard::draw_dashboard(ux_window& win, rect& r)
                 { pos.x + r.w, pos.y + get_height() }, ImColor(dark_sensor_bg));
 
     auto size = ImGui::CalcTextSize(name.c_str());
-    ImGui::SetCursorPos(ImVec2( r.w / 2 - size.x / 2, 5 ));
+    auto collapse_buton_h = 28 + 3;
+    ImGui::SetCursorPos(ImVec2( r.w / 2 - size.x / 2, 5.f + collapse_buton_h));
```

Comment:
Why use float? before it was not so. You could have made `collapse_buton_h` float if needed.

Reply:
I added calculation and IDE showed a warning so I added float.